cfme/markers/env.py

```diff
@@ -56,26 +84,22 @@ class TCPEnvironmentMarker(EnvironmentMarker):
     CHOICES = ['ipv4', 'ipv6']
 
 
-class PodifiedEnvironmentMarker(EnvironmentMarker):
-    """Podified Appliance Environment Marker"""
-    NAME = 'podified'
-    CHOICES = ['pod', 'vm']
-
-
 def pytest_configure(config):
     config.addinivalue_line('markers', 'provider: Mark a test function/class/module for a provider')
     config.addinivalue_line('markers', 'uses_testgen: Marker placed on tests that use testgen')
-    for envmark in [BrowserEnvironmentMarker, TCPEnvironmentMarker, PodifiedEnvironmentMarker]:
+    from cfme.markers.env_markers.appliance import ApplianceEnvironmentMarker
+    for envmark in [ApplianceEnvironmentMarker, BrowserEnvironmentMarker, TCPEnvironmentMarker]:
         config.addinivalue_line('markers', envmark.marker_description())
 
 
 def pytest_generate_tests(metafunc):
     from cfme.markers.env_markers.provider import ProviderEnvironmentMarker
+    from cfme.markers.env_markers.appliance import ApplianceEnvironmentMarker
```

Comment:
Can we  import this at the top so as to avoid importing it again ?

Reply:
Would create a circular import at the module level - cfme/markers/env_markers/appliance.py module imports from cfme/markers/env.py

Reply:
@sshveta, Mike is right. see above answer